Formula/qwt.rb

```diff
@@ -62,21 +62,21 @@ def install
       system ENV.cxx, "test.cpp", "-o", "out",
         "-std=c++11",
         "-framework", "qwt", "-framework", "QtCore",
-        "-F#{lib}", "-F#{Formula["qt@5"].opt_lib}",
+        "-F#{lib}", "-F#{Formula["qt"].opt_lib}",
         "-I#{lib}/qwt.framework/Headers",
-        "-I#{Formula["qt@5"].opt_lib}/QtCore.framework/Versions/5/Headers",
-        "-I#{Formula["qt@5"].opt_lib}/QtGui.framework/Versions/5/Headers"
+        "-I#{Formula["qt"].opt_lib}/QtCore.framework/Versions/5/Headers",
+        "-I#{Formula["qt"].opt_lib}/QtGui.framework/Versions/5/Headers"
```

Comment:
```suggestion
        "-I#{Formula["qt"].opt_lib}/QtCore.framework/Versions/6/Headers",
        "-I#{Formula["qt"].opt_lib}/QtGui.framework/Versions/6/Headers"
```
There's nothing wrong with `Formula["qt"].opt_lib`. You just failed to change the `5`s to `6`s.

Reply:
Thanks!